plugin.video.wizjatv/main.py

```diff
@@ -6,25 +6,28 @@
 PY2 = sys.version_info[0] == 2
 if PY2:
     from urlparse import parse_qs
+    from urllib import urlencode
 else:
-    from urllib.parse import parse_qs
+    from urllib.parse import parse_qs, urlencode
 
 from ptw.debug import log_exception, log, start_trace, stop_trace, TRACE_ALL
 import wizja
 
+# Addon Internal Modes:
+PLAY_LINK = 6
+
 obj = wizja.WizjaTvApi()
 s = requests.Session()
 
 def CATEGORIES():
     WizjaTV()
 
-def addDir(name, url, mode, thumb, fanart, opis, isFolder=True, total=1):
-    u=sys.argv[0]+'?url='+urllib.quote_plus(url)+'&mode='+str(mode)+'&name='+urllib.quote_plus(name)
-    ok = True
+def addDir(name, url, mode, thumb, fanart='', opis='', isFolder=True, total=1):
```

Comment:
Python pozawala na nazywanie argumentów, w tym używanie domyślnych wartości. Warto z tego mocno korzystać (tak jak ostatnie dwa argumenty). Jeśli damy więcej domyślnych wartości to można będzie ich nie używać w wywołaniu zamiast pisać szereg `None` czy `''` (*patrz niżej*).

Reply:
O widzisz i tu wychodzi moja amatorka :D nie wiedziałem, że tak można :P